src/Storage/DBFile.php

```diff
@@ -403,7 +404,9 @@ public function getAbsoluteSize()
      */
     public function setOriginal($original)
     {
-        $this->failover = $original;
+        if ($original instanceof ViewableData) {
```

Comment:
This change seems unnecessary and also contradicts the PHPDoc which says the `$original` param should be an `AssetContainer`.

Reply:
I don't see how the type check has anything to do with using the setter or not?

Reply:
Because you can't set a non-viewabledata object as a failover?

Reply:
1. Is that true? Where are those limtations enforced?
2. The PHPDoc is thus wrong and needs correcting
3. This is a risky / major change as if someone had passed the documented type through (or any duck-typed object) then it would now break...

I'm just struggling to see how this change provides a defined benefit